requirements.txt

```diff
@@ -0,0 +1,72 @@
+# This file is @generated by PDM.
+# Please do not edit it manually.
+
+annoy==1.17.3
+arrow==1.3.0
+certifi==2024.8.30
+charset-normalizer==3.3.2
+click==8.1.7
+codecarbon==2.7.1
+colorama==0.4.6; platform_system == "Windows"
+contourpy==1.3.0
+copairs @ git+https://github.com/cytomining/copairs.git@880f22a551bd897896d148a0b07baa99d981c6a9
+cycler==0.12.1
+drjit==0.4.6
+fonttools==4.54.1
+greenlet==3.1.1; (platform_machine == "win32" or platform_machine == "WIN32" or platform_machine == "AMD64" or platform_machine == "amd64" or platform_machine == "x86_64" or platform_machine == "ppc64le" or platform_machine == "aarch64") and python_version < "3.13"
+idna==3.10
+joblib==1.4.2
+kaleido==0.2.1.post1
+kiwisolver==1.4.7
+llvmlite==0.43.0
+markdown-it-py==3.0.0
+matplotlib==3.9.2
+mdurl==0.1.2
+mitsuba==3.5.2
+numba==0.60.0
+numpy==1.24.4
+packaging==24.1
+pacmap==0.7.3
+pandas==2.2.3
+patsy==0.5.6
+pillow==10.4.0
+platformdirs==4.3.6
+plotly==5.24.1
+pooch==1.8.2
+prometheus-client==0.21.0
+prompt-toolkit==3.0.36
+psutil==6.0.0
+py-cpuinfo==9.0.0
+pycytominer @ git+https://github.com/cytomining/pycytominer.git@36241269c4293c24484986568ca16b2d7eb9e808
+pygments==2.18.0
+pymcubes==0.1.6
+pynvml==11.5.3
+pyparsing==3.1.4
+python-dateutil==2.9.0.post0
+pytz==2024.2
+pyvista==0.44.1
+questionary==2.0.1
+rapidfuzz==3.10.0
+requests==2.32.3
+rich==13.9.2
+scikit-learn==1.5.2
+scipy==1.8.1
+scooby==0.10.0
+seaborn==0.13.2
+shellingham==1.5.4
+six==1.16.0
+sqlalchemy==2.0.35
+statsmodels==0.14.4
+tenacity==9.0.0
+threadpoolctl==3.5.0
+torch-scatter==2.1.2
```

Comment:
seems a bit off that torch is not a dependency? is this correct?

Reply:
Yes, this is something I need to fix. In the updated .toml file, the build system flag has disappeared (torch needs to be included there to be included in the requirements.txt file ig - maybe @pgarrison can clarify this)

Reply:
I don't think it is related to the build system section of pyproject.toml: that section only specified numpy and pdm, nothing about torch. I think it's more likely related to leaving out cyto-dl

Reply:
I tried to install the env locally to see if that worked and pdm gave me an error citing that torch had to be included in the build section!

Reply:
also, cyto-dl is included in the requirements.txt